src/EventService.Broker/Requests/UserService.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using LT.DigitalOffice.EventService.Broker.Requests.Interfaces;
+using LT.DigitalOffice.Kernel.BrokerSupport.Helpers;
+using LT.DigitalOffice.Models.Broker.Common;
+using LT.DigitalOffice.Models.Broker.Models;
+using LT.DigitalOffice.Models.Broker.Requests.User;
+using LT.DigitalOffice.Models.Broker.Responses.User;
+using MassTransit;
+
+namespace LT.DigitalOffice.EventService.Broker.Requests;
+
+public class UserService : IUserService
+{
+  private readonly IRequestClient<ICheckUsersExistence> _rcCheckUserExistence;
+  private readonly IRequestClient<IGetUsersDataRequest> _rcGetUsersData;
+
+  public UserService(
+    IRequestClient<ICheckUsersExistence> rcCheckUserExistence, 
+    IRequestClient<IGetUsersDataRequest> rcGetUsersData)
+  {
+    _rcCheckUserExistence = rcCheckUserExistence;
+    _rcGetUsersData = rcGetUsersData;
+  }
+
+  public async Task<List<Guid>> CheckUsersExistenceAsync(List<Guid> usersIds, List<string> errors = null)
```

Comment:
You can return Task<bool>. According to method naming

Reply:
mm.. consumer in UserSerice returns `List<Guid>`

Reply:
OK, I got it
but after getting response, can write 

```
var existingUserIds = (await RequestHandler.ProcessRequest<ICheckUsersExistence, ICheckUsersExistence>(
        _rcCheckUserExistence,
        ICheckUsersExistence.CreateObj(usersIds),
        errors))
      ?.UserIds
      
return userIds == existingUserIds;
```